src/eos_gates.js

```diff
@@ -207,10 +207,26 @@ var Application = new Lang.Class({
                                 max_width_chars: 30,
                                 label: this.getHelpMessage() });
         label.get_style_context().add_class('unsupported-subtitle');
-        label.connect('activate-link', Lang.bind(this, function() {
+        label.connect('activate-link', (label, uri) => {
+            if (uri !== 'x-eos-gates:launch-app-center')
```

Comment:
Nit: Could 'x-eos-gates:launch-app-center' be a constant on top of the file instead?

Reply:
It could, thanks. I'll fix that & merge tomorrow.